kvm-libvirt/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -0,0 +1,33 @@
+module "bootkube" {
+  source = "github.com/kinvolk/terraform-render-bootkube?ref=f46efe1427f8d35c01da64fde95a798778cbe31f"  # currently in PR #12
```

Comment:
Need to update the `ref` for https://github.com/kinvolk/terraform-render-bootkube/pull/12

Reply:
Please also use `git::https`